cmd/rookflex/cmd/expander.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/rook/rook/pkg/daemon/ceph/agent/flexvolume"
+	"github.com/spf13/cobra"
+	"os/exec"
+	"strconv"
+)
+
+var (
+	expandFSCmd = &cobra.Command{
+		Use:   "expandfs",
+		Short: "Expands the size of pod filesystem",
+		RunE:  handleExpandFs,
+	}
+)
+
+func init() {
+	RootCmd.AddCommand(expandFSCmd)
+}
+
+func handleExpandFs(cmd *cobra.Command, args []string) error {
+	var opts = &flexvolume.ExpandOptions{}
+
+	err := json.Unmarshal([]byte(args[0]), opts)
+	if err != nil {
+		return fmt.Errorf(
+			"Rook: Could not parse options for expand %s. Got %v",
+			args[1],
+			err,
+		)
+	}
+	client, err := getRPCClient()
+	if err != nil {
+		return fmt.Errorf("Rook: Error getting RPC client: %v", err)
+	}
+
+	size, err := strconv.ParseInt(args[3], 10, 64)
+	if err != nil {
+		return fmt.Errorf("Rook: Error while decoding RBD size: %v", err)
+	}
+
+	err = client.Call(
```

Comment:
This call looks like it could pretty easily be one line. If it doesn't neatly fit in a 100-ish columns, I would prefer to see the expandargs placed in a variable and that used as a parameter rather than breaking this across so many lines. Breaking this short function across so many lines unnecessarily adds the impression more cyclomatic complexity to the code.

Reply:
I understand. I will change it.